### ISSUE TITLE
"Play Youtube Video" aria-label should be translatable

### ISSUE THREAD

**Issue by audrasjb** [wp-media/wp-rocket#7202]
**Describe the bug**
When using image remplacement for Youtube Iframes, the "Play Youtube Video" aria-label should be translatable. This is currently an issue with Accessibility Guidelines and as far as I know there is now way to properly override this string yet.

**To Reproduce**
1. Activate lazyloading for video and activate thumbnails images for Youtube
2. Switch your installation to another locale, like fr_FR (Français)
3. Add a video on a post and confirm that the "Play Youtube Video" aria-label is not translated

**Expected behavior**
"Play Youtube Video" should be translated.

- You have to replace the current implementation to a regular gettext string so it can be translated
- Ideally, this string could be filterable via a hook (not mandatory)

**Desktop (please complete the following information):**
 - OS: All
 - Browser: All
 - Version: current

**Smartphone (please complete the following information):**
Same

**Additional context**
The issue lies here: https://github.com/wp-media/wp-rocket/blob/develop/inc/Dependencies/RocketLazyload/Assets.php#L283

Cheers Rocketeers!
Jb
